packages/@atjson/renderer-commonmark/src/index.ts

```diff
@@ -26,31 +26,60 @@ import {
 } from "./lib/punctuation";
 export * from "./lib/punctuation";
 
+function getStart(a: { start: number }) {
+  return a.start;
+}
+
+function getEnd(a: { end: number }) {
+  return a.end;
+}
+
+function isParseAnnotation(a: Annotation<any>) {
+  let constructor = a.constructor as AnnotationConstructor<
+    typeof a,
+    typeof a.attributes
+  >;
+  return constructor.vendorPrefix === "atjson" && a.type === "parse-token";
+}
+
+function isParseOrUnknown(a: Annotation<any>) {
```

Comment:
Same as the above comment, this could be written as:

```ts
function isParseAnnotation<T extends Annotation>(annotation: T): annotation is ParseAnnotation | UnknownAnnotation {
```